.travis.yml

```diff
@@ -1,7 +1,12 @@
 language: haskell
+before_install:
+  - sudo apt-get update -qq
+  - sudo apt-get install -y aspell aspell-en  
+
 install:
     - cabal install mtl
     - sudo apt-get install arduino-core
     - git clone https://github.com/sudar/Arduino-Makefile.git
 script:
     - ./test
+    - aspell --extra-dicts ./frp.en.pws list < README.md
```

Comment:
Looks like the path to the extra dict is wrong. Should be and extra dot infront.